apps/remix-ide/src/blockchain/execution-context.js

```diff
@@ -146,7 +143,7 @@ export class ExecutionContext {
         this.askPermission()
         this.executionContext = context
         web3.setProvider(injectedProvider)
-        this._updateBlockGasLimit()
+        this._updateChainContext()
```

Comment:
No need to wait for this? because it affects currentfork?

Reply:
good point, will check that asap

Reply:
That works ok like this, but it's better to `await`. please see the pr https://github.com/ethereum/remix-project/pull/1340